core/arch/arm/mm/mobj.c

```diff
@@ -155,12 +157,6 @@ struct mobj *mobj_phys_alloc(paddr_t pa, size_t size, uint32_t cattr,
 	enum teecore_memtypes area_type;
 	void *va;
 
-	if ((pa & CORE_MMU_USER_PARAM_MASK) ||
```

Comment:
@jenswi-linaro I use `mobj_phys` to represent buffer in static SHM. This buffers are not page-aligned. So, I either should remove this check, or add another `mobj` type.